docs/src/dataDisplay/table/tableSampleOnClick.jsx

```diff
@@ -1,26 +1,20 @@
-import React from 'react';
-import {
-    Header,
-    Table,
-} from 'react-cm-ui';
-import Highlighter from '../../global/highlighter';
-
-const TableHeaderCellOnClickSample = `import React from 'react';
 import { Table } from 'react-cm-ui';
+import React from 'react';
 
-export default class TableHeaderCellOnClickSample extends React.Component {
+class TableSampleOnClick extends React.PureComponent {
     constructor(props) {
         super(props);
+
         this.onClick = this.onClick.bind(this);
     }
 
     onClick() {
-        // ... do whatever the clicks are supposed to do ...
+        window.alert('Clayton Kershaw!'); // eslint-disable-line no-alert
```

Comment:
Ah yes, good ol' Clayton Kershaw!  Didn't you and Ryan Meyer used to tussle between Clayton Kershaw and Madison Bumgarner back in the day? ;-)

Reply:
HAHAHA! Good times.